api/src/utils/media/media-utils.ts

```diff
@@ -114,3 +118,59 @@ export const isZipMimetype = (mimetype: string): boolean => {
     regex.test(mimetype)
   );
 };
+
+export const parseUnknownMediaToDwCAArchive = (unknownMedia: UnknownMedia): DWCArchive => {
+  const parsedMedia = parseUnknownMedia(unknownMedia);
+
+  if (!parsedMedia) {
+    throw new ApiGeneralError('Failed to parse submission', [
+      'DarwinCoreService->prepDWCArchive',
+      'unknown media file was empty or unable to be parsed'
+    ]);
+  }
+
+  if (!(parsedMedia instanceof ArchiveFile)) {
+    throw new ApiGeneralError('Failed to parse submission', [
+      'DarwinCoreService->prepDWCArchive',
+      'unknown media file was not a valid Archive file'
+    ]);
+  }
+
+  return new DWCArchive(parsedMedia);
+};
+
+export const parseEMLtoJSONSource = (eml: EMLFile): any => {
+  const options = {
+    ignoreAttributes: false,
+    attributeNamePrefix: '@_',
+    parseTagValue: false, //passes all through as strings. this avoids problems where text fields have numbers only but need to be interpreted as text.
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I think eslint comments are discouraged. Should the underline prefix make this complaint go away anyway?